types/koa/test/index.ts

```diff
@@ -47,6 +47,23 @@ app.use((ctx, next) => {
     });
 });
 
+app.use(ctx => {
+    ctx.accepts(); // $ExpectType string[]
```

Comment:
I haven't seen this `$ExpectType` reference before, a quick google search made reference to an eslint plugin.

Have you verified this test is working as expected?

Reply:
This is how type assertions work in dtslint, which is the tool used to test types in DefinitelyTyped. https://github.com/DefinitelyTyped/DefinitelyTyped#running-tests

There are 3 types of assertions:

1. Making sure TypeScript doesn’t complain
2. Assert a specific type using `// $ExpectType`
3. Assert a type doesn’t match using `// $ExpectError`

I hope this helps you write type definitions in the future. :)